manifests/l/Liquit/Workspace/PowerShell42/4.2.3735.5913/Liquit.Workspace.PowerShell42.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Liquit.Workspace.PowerShell42
+PackageVersion: 4.2.3735.5913
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{2E329CAA-7BD3-4D66-845D-2AA3BDF78771}'
+AppsAndFeaturesEntries:
+- DisplayName: Liquit Workspace PowerShell
+  Publisher: Liquit Software B.V.
+  DisplayVersion: 4.2.3735.5913
+  ProductCode: '{2E329CAA-7BD3-4D66-845D-2AA3BDF78771}'
+  UpgradeCode: '{A8A90B86-A5CB-46BA-8E01-BFD50757157D}'

```

```suggestion
- ProductCode: '{2E329CAA-7BD3-4D66-845D-2AA3BDF78771}'
  UpgradeCode: '{A8A90B86-A5CB-46BA-8E01-BFD50757157D}'
```
`DisplayVersion` should be omitted when it is the same as `PackageVersion`
`DisplayName` should be omitted when it is the same as `PackageName`
`Publisher` should be omitted when it is the same as `Publisher` from the defaultLocale manifest

---

why do you suggest to use two ProductCode and UpgradeCode entries?

---

> why do you suggest to use two ProductCode and UpgradeCode entries?

I don't, that's just how the suggestion appears on GitHub Web. It won't actually create two